### DISCUSSION TITLE
A Question on SQL

### DISCUSSION THREAD

**Discussion by Tofunmi-I** [community/community#109730] [Programming Help]
I'm using a JOIN statement to merge EmployeeID, First Name, and Last Name from Table 1 with Job Title and Salary from Table 2. 
Using a CASE statement, I'm adjusting salaries after a raise, with a specific percentage for 'salesman' job title.

How can I make 'salesman' job title employees appear first in the result column, followed by other job titles? I want to emphasize their contribution with a 50% raise. Open to suggestions and best practices. Thanks!

**Comment by namith1003** [comment]
```
 SELECT 
     EmployeeID,
     FirstName,
     LastName,
     JobTitle,
     CASE 
         WHEN JobTitle = 'salesman' THEN Salary * 1.5 
         ELSE Salary
     END AS AdjustedSalary
 FROM 
     Table1
 JOIN 
     Table2 ON Table1.EmployeeID = Table2.EmployeeID
 ORDER BY 
     CASE 
         WHEN JobTitle = 'salesman' THEN 0
         ELSE 1
     END;
```
     
     
when you order in ascending order, and assign 0 only when it is salesman then all the salesman rows will be at the beginning followed by anything else as you asked, you may try this and if this is the answer then all good if not please add on to the request.

I think you had asked this before as well.

**Reply by Tofunmi-I** [reply]
It did work. Thanks

**Comment by ghostinhershell** [comment]
Hi @Tofunmi-I,

The GitHub Community on Discussions is a space for all GitHub users to come and discuss ideas and projects. Your post was removed because it contained content that violates our Community Code of Conduct, including posting the same comment in multiple places. As a result, we have hidden the duplicate comments, and we ask that you no longer cross-post.

For additional guidance, please review our [Community on Discussions Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md) and [GitHub’s Terms of Service](https://docs.github.com/en/github/site-policy/github-terms-of-service).

**Reply by Tofunmi-I** [reply]
I sincerely apologize for that. I am new to GitHub and still trying to navigate my way around it.

**Comment by Jeesejack** [comment]
perfect

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @Tofunmi-I ! 

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.

**Comment by WillTheFarmer** [comment]
Why is this thread still marked Open and Unanswered?
